app/src/main/java/com/android/partagix/model/ItemViewModel.kt

```diff
@@ -83,6 +83,10 @@ class ItemViewModel(
     }
   }
 
+  fun check(id: String, userId: String?): Boolean {
```

Comment:
I think it would be great to
- add a little line of documentation
- choose a more explicit function name, for example: `compareIDs`